src/test/java/fr/insee/pogues/utils/PoguesModelUtilsTest.java

```diff
@@ -0,0 +1,42 @@
+package fr.insee.pogues.utils;
+
+import fr.insee.pogues.exception.IllegalFlowControlException;
+import fr.insee.pogues.exception.IllegalIterationException;
+import fr.insee.pogues.model.DynamicIterationType;
+import fr.insee.pogues.model.FlowControlType;
+import fr.insee.pogues.model.IterationType;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class PoguesModelUtilsTest {
+
+    @Test
+    void getFlowControlBounds_nullIfTrue() {
```

Comment:
method name does not reflect what it does

Reply:
The property name is actually confusing here, I added a comment